src/site.settings.php

```diff
@@ -25,7 +28,7 @@
     $argument = $context['argv'][1];
 
     if ($argument === 'config:export' || $argument === 'cex' || $argument === 'config-export') {
-      $configSplit = SiteSettings::getConfigSplitArray(SiteSettings::getRoboConfig());
+      $configSplit = SiteSettings::getConfigSplitArray($configFileReader->getRoboConfig());
```

Comment:
This could actually live inside the ConfigFileReader as well. No need for referencing Site Settings thaen

Reply:
Done